Source/Test/Rewriting/RewritingEngine.cs

```diff
@@ -198,19 +204,19 @@ private void Run()
             // Create the output directory and copy any necessary files.
             string outputDirectory = this.CreateOutputDirectoryAndCopyFiles();
 
-            this.Pending = new Queue<string>();
-
-            int errors = 0;
-            // Rewrite the assembly files to the output directory.
-            foreach (string assemblyPath in this.Options.AssemblyPaths)
+            // Create list of all assemblies we need to transform.
+            this.AssemblyNameScope = new HashSet<string>();
```

Comment:
And I guess it doesn't really matter the order of assemblies that we rewrite, right? We only need to care is that when we rewrite an assembly (in whatever order) that these X other assemblies are also going to get rewritten (or already have)?